t/50reverse-proxy-early-response.t

```diff
@@ -421,12 +422,18 @@ sub read {
     $buf;
 }
 
+# This code is originated from socketpool.c
+# Find for the comment "test if the connection is still alive".
 sub is_alive {
     my ($self) = @_;
     return undef unless $self->{sock};
     my $buf;
+    #local $! = 0;
```

Comment:
you don't need this line?

Reply:
Looks like it doesn't affect the result. Removed 6b92d2c